docs/Creating-Components.md

```diff
@@ -0,0 +1,149 @@
+# How to create a new component
+There are a lot of moving parts to Simorgh. We use the [Presentational & Container pattern](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0) for our components, which we build in a few stages.
+
+## Part 1: Create the presentational component
+Run the storybook (`npm run storybook`) and visit http://localhost:9001.
+
+Add your component in `src/app/components/[Component Name]/`. You will need:
+
+* index.jsx - defines the component
+* index.stories.jsx - defines variations of the component.
+
+Work locally within these two files - the Storybook pattern library will be updated automatically with hot module reloading - to create the component front-end you need.
+
+NB: we try to build "AMP-first", but this is difficult in practice because we are unable to inject the AMP dependency into Storybook. For now, check the AMP output manually in Part 2 of the process. Simorgh currently only supports one 'view' of components; components which require a different implementation for AMP must wait for the outcome of our [AMP/canonical component investigation](https://github.com/BBC-News/simorgh/issues/884).
+
+Once done, make a PR - this first step will need code review and UX QA.
+
+Example implementations below:
+
+### index.jsx
+```js
+import styled from 'styled-components';
+
+const Blockquote = styled.blockquote`
+  background: #f9f9f9;
+`;
+
+export default Blockquote;
+```
+
+### index.stories.jsx
+```js
+import React from 'react';
+import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
+import Blockquote from './index';
+
+storiesOf('Blockquote', module).add('default - quote only', () => <Blockquote>To be or not to be, that is the question</Blockquote>);
+storiesOf('Blockquote', module).add('quote with cite', () => (
+  <Blockquote cite="William Shakespeare">To be or not to be, that is the question</Blockquote>
+));
+```
+
+## Part 2: Create the container component
```

Comment:
Feels like this and line 108 (`### Add a container`) are the same thing. Maybe this heading and summary is in the wrong place?